client/go/commands/usage_templates.go

```diff
@@ -92,3 +92,59 @@ Example:
     heketi -server http://localhost:8080 cluster list
 
 `
+var usageTemplateNode = `Node is a command used for managing Heketi nodes.
+
+Usage:
+    heketi -server [server] [options] node [subcommand]
+
+The subcommands are:
+    add        Adds a new node to the designated cluster.
+    info       Returns information about the specific node.
+    destroy    Destroys node with specified id.
+
+Use "heketi node [subcommand] -help" for more information about a subcommand
+
+`
+var usageTemplateNodeAdd = `node add is a command used for adding a node to the specified cluster.
+
+Usage:
+    heketi -server [server] [options] node add [flags]
+
+The flags are:
+    -zone                  The zone in which the node should reside.
```

Comment:
Ok, now I see how it works.  There is an API called PrintDefaults() which prints the flags out to stderr if flag.SetOutput() hasn't been specified.  I have created a [gist](https://gist.github.com/lpabon/ab51a9a95a2b506a2b6d) which prints the following out:

```
$ go run main.go node add -h

Node add is a command used for adding a node to the specified cluster.

USAGE
    heketi node add [options]

OPTIONS
  -cluster="": The cluster in which the node should reside
  -managment-host-name="": Managment host name
  -storage-host-name="": Storage network host name
  -zone=0: The zone in which the node should reside

EXAMPLES
	Add a node to heketi
	    $ heketi -server http://localhost:8080 node add \
	    	-zone 3 \
	    	-cluster 3e098cb4407d7109806bb196d9e8f095 \
	    	-managment-host-name node1-manage.gluster.lab.com \
	    	-storage-host-name node1-storage.gluster.lab.com
exit status 2
```